### ISSUE TITLE
Header: Too large to fit on 1 screen

### ISSUE THREAD

**Issue by kaylarep** [opencollective/foundation-website#47]
Ideally the entire header + subheader + complete image would be visible on 1 screen. 

It currently isn't + the image is cut- you have to scroll immediately to figure out what's going on here. Changing my zoom to 75% puts it all on 1 page.

![image](https://user-images.githubusercontent.com/64920686/175178806-c6d1e40e-791a-4378-98d7-fdbc8c952da3.png)

**Comment by jboolean** [comment]
What screen height is this?

A small desktop screen would be 1024x768 which looks like this

![localhost_8080_ (1)](https://user-images.githubusercontent.com/406931/175576012-bc71bce1-20aa-4bbb-bdbb-272f213760ad.png)

**Comment by kaylarep** [comment]
@jboolean my screen size is 1003 × 564

**Comment by Memo-Es** [comment]
One thing I was thinking about the header @jboolean: We should fix the height, so it doesn't 'jump' if the length of the Collective name varies. I also think we could do better at nailing down the right breakpoints so the issue described here and [this other one](https://d.pr/i/nrzbXn) is solved

**Comment by kaylarep** [comment]
@Memo-Es 
My screen size is 1003 × 564

Here's some stats about my OS, let me know if you need any different info: 
![image](https://user-images.githubusercontent.com/64920686/176262944-371d6719-8d69-4040-94d5-780969890aae.png)

**Comment by jboolean** [comment]
I'm going to bounce this back to @Memo-Es because it's a very open ended design question.

**Comment by Memo-Es** [comment]
Yeah I'm not happy with the current header, I will take a stab at it and update it asap
